scalactic-test/src/test/scala/org/scalactic/AccumulationSpec.scala

```diff
@@ -30,7 +30,7 @@ class AccumulationSpec extends UnitSpec with Accumulation with TypeCheckedTriple
   def parseDate(in: String): Date Or One[ErrorMessage] = {
     val regex = "\\d\\d\\d\\d-\\d\\d?-\\d\\d?".r
     if (regex.pattern.matcher(in).matches()) {
-      val tokens = in.split("-")
+      val tokens = in.split("-").map(_.nn)
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.